### PR TITLE
Notebook Explaining Centrality Algorithms solving issue #5535 from networkx Issues

### PR DESCRIPTION
**Pull request by 0ddoes**
Sir, I tried to convert it to .md file. However the Output Figures are not shown in the file. Can you help me to correct it. It would be nice to have a contribution guide to nx_guides. 😅

### CODE REVIEW

File: content/exploratory_notebooks/Centrality.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+import matplotlib.pyplot as plt
+import random as ran
+import numpy as np
+import seaborn as sns
+import pandas as pd
+```
+:::
+
+::: {.cell .code execution_count="4"}
+``` {.python}
```
Comment (MridulS) [inline content/exploratory_notebooks/Centrality.md:37]: How are you converting your notebooks to markdown?

These blocks of code shouldn't be there.

Comment (MridulS) [inline content/exploratory_notebooks/Centrality.md:37]: How are you converting your notebooks to markdown?

These blocks of code shouldn't be there.

**Comment by MridulS** [comment]
The output figures can't be embedded into the markdown notebook and that's fine! The notebook is supposed to be executed and create the figures using github actions, which is then pushed to the nx-guides website.

**Comment by 0ddoes** [comment]
Sir, there was an online converter for jupyter notebooks to markfown which I used. Is there a better way?

**Comment by 0ddoes** [comment]
Sir, here the errors are that it cannot recognise the ```import seaborn as sns``` and  access the ```polblogs.gml``` file that I have added in the data folder. Subsequent errors are due to same issue. How can I actually correct this? Please let me know.

**Comment by 0ddoes** [comment]
@MridulS Can you please help me with how I can correct this?

**Comment by rossbar** [comment]
> Can you please help me with how I can correct this?

The CI was showing two failures: one due the fact that the notebook was not included in the site anywhere, and another due to the fact that heading levels were skipped. These are common problems so I went ahead and pushed up fixes for each. That should at least clear up the current CI failures... we'll see if we hit more now :)

**Comment by MridulS** [comment]
@0ddoes we would like to show the reasons and explain the algorithms in nx-guides, not only use the networkx functions to get the results. So while doing analysis of datasets we should be also looking at the results of the function calls and explain them. I'll go ahead and close this PR. If you have the time and interest please feel free to submit a new pull request :)

**Pull request closed without merging**